composer.json

```diff
@@ -0,0 +1,5 @@
+{
+    "require": {
+        "symfony/symfony": "^2.7"
```

Comment:
You should require only the Symfony `Console` component - no need to grab the full-stack framework.